RetailCoder.VBE/UI/SelectionChangeService.cs

```diff
@@ -123,7 +123,7 @@ private void DispatchSelectedDesignerDeclaration(IVBComponent component)
                 return;
             }
             var form =
-                _parser.State.DeclarationFinder.UserDeclarations(DeclarationType.UserForm).SingleOrDefault(decl =>
+                _parser.State.DeclarationFinder.UserDeclarations(DeclarationType.ClassModule).SingleOrDefault(decl =>
```

Comment:
This should probably use `MatchName` first and then apply the other criteria.

Reply:
Was check a quickfix along the way, didn't wanna go messing too much. Happy to revisit tho.